samples/central-login/src/main.js

```diff
@@ -0,0 +1,99 @@
+import * as forgerock from '@forgerock/javascript-sdk';
+
+/*
+ * @forgerock/javascript-sdk
+ *
+ * index.html
+ *
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+const FATAL = 'Fatal';
+
+forgerock.Config.set({
+  clientId: '<Your OAuth Client>', // e.g. 'ForgeRockSDKClient'
+  redirectUri: '<Your Redirect/Callback URL>', // e.g. 'https://sdkapp.example.com:8443/central-login/'
+  scope: '<Your OAuth Scopes>', // e.g. 'openid profile me.read'
+  serverConfig: {
+    baseUrl: '<Your AM URL>', // e.g. 'https://openam.example.com:9443/openam/'
+    timeout: 3000, // 3000 to 5000 is good, this impacts the redirect time to login
+  },
+  realmPath: '<Your Realm>', // e.g. 'root'
+});
```

Comment:
If we are instructing the developer to use the `.env` file, how are the values being interpolated into this file?

Reply:
webpack config applies `webpack.definePlugin` just like react app.

Reply:
huh, did i not push this. checking.

Reply:
push i guess failed, should be updated now.